helm/ako/values.yaml

```diff
@@ -29,6 +29,11 @@ AKOSettings:
   servicesAPI: false # Flag that enables AKO in services API mode: https://kubernetes-sigs.github.io/service-apis/. Currently implemented only for L4. This flag uses the upstream GA APIs which are not backward compatible 
                      # with the advancedL4 APIs which uses a fork and a version of v1alpha1pre1 
   vipPerNamespace: "false" # Enabling this flag would tell AKO to create Parent VS per Namespace in EVH mode
+  # This is the list of system namespaces from which AKO will not listen any Kubernetes or Openshift object event.
+  blackListedNamespaces: []
```

Comment:
can this also be some kind of filter for excluded namespaces, rather than a list, say based on some annotations ?

Reply:
I did not find any system specific annotation in openshift or kubernetes that can be used here. Using namespace based filter (namespace sync feature) is another approach for newly created user namespaces.